### PR TITLE
[Bugfix] - Fix bug when FCE Paragraph with #

### PR DESCRIPTION
**Pull request by ivandrofly**
Do not merge yet, I'll add some test ;)

### CODE REVIEW

**Comment by ivandrofly** [comment]
Done :smile:  /cc @niksedk

File: libse/Forms/FixCommonErrors/FixMusicNotation.cs [hunk 1]
```diff
@@ -17,11 +17,30 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
...
                     {
-                        newText = newText.Replace(musicSymbol, Configuration.Settings.Tools.MusicSymbol);
-                        newText = newText.Replace(musicSymbol.ToUpper(), Configuration.Settings.Tools.MusicSymbol);
+                        if (containsFontTag && musicSymbol == "#")
+                        {
+                            var idx = newText.IndexOf('#');
+                            while (idx >= 0)
+                            {
+                                // <font color="#808080">NATIVE HAWAIIAN CHANTING</font>
+                                var isInsideFontTag = (idx < 13) ? true : (newText[idx - 1] == '"' && (newText.Length > idx + 2 && char.IsDigit(newText[idx + 1]) && char.IsDigit(newText[idx + 2])));
```
Comment (xylographe) [inline libse/Forms/FixCommonErrors/FixMusicNotation.cs]: Shouldn't that be `(idx < 13) ? false : ` ?

Comment (ivandrofly) [inline libse/Forms/FixCommonErrors/FixMusicNotation.cs]: > Shouldn't that be (idx < 13) ? false : ?

:+1:

Comment (xylographe) [inline libse/Forms/FixCommonErrors/FixMusicNotation.cs]: Also, I think, `#` is followed by hexadecimal digits:
 `Uri.IsHexDigit(newText[idx + 1]) && Uri.IsHexDigit(newText[idx + 2])`

Comment (ivandrofly) [inline libse/Forms/FixCommonErrors/FixMusicNotation.cs]: > Also, I think, # is followed by hexadecimal digits:
> Uri.IsHexDigit(newText[idx + 1]) && Uri.IsHexDigit(newText[idx + 2])

Nice catch :+1:

**Comment by niksedk** [comment]
thx :)

**Pull request merged**